.github/CONTRIBUTING.md

```diff
@@ -27,17 +27,17 @@ If you write and contribute a full topic, we will add your name (or your company
 
 ## Contribution guidelines
 
-We use [Markdown](http://daringfireball.net/projects/markdown/) to write our documentation, which is a simple markup language that we convert to HTML using [Kramdown](http://kramdown.gettalong.org/syntax.html). Check [Templates](#templates) for examples of styles and markdown.
+Write content using [Kramdown](https://kramdown.gettalong.org/), which is a simple markup language. We use Kramdown, Liquid, and [Jekyll](https://jekyllrb.com/) to generate a static site hosted through [GH Pages](https://help.github.com/articles/what-is-github-pages/). Check [Templates](#templates) for examples of styles and markdown.
 
 You can update existing or add new topics in their respective Magento 2 versioned directories (2.1, 2.2, 2.3, and onward). If you need help finding a directory for your content, we can help in your PR.
 
 The following guidelines may answer most of your questions and help you get started:
 
 1.  Check [existing pull requests](https://GitHub.com/magento/devdocs/pulls) and make sure you are not duplicating work!
 
-1.  For large contributions or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
+2.  For large contributions or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
```

Comment:
@lorikrell, numbers don't need to be sequential in markdown to render sequentially in HTML. In fact, it's easier to maintain non-sequential numbered lists in markdown so that you don't have to manually re-rnumber the list if you need to add/delete a list item.

Reply:
Sorry for the sequential numbering getting in there. The new IDE I'm using adding that automatically. I'll double check the configs. Thanks!